tools/v8-prof/tick-processor.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+var cp = require('child_process');
+var fs = require('fs');
+var path = require('path');
+
+var toolsPath = path.join(__dirname, '..', '..', 'deps', 'v8', 'tools');
+var scriptFiles = [
+    path.join(__dirname, 'polyfill.js'),
+    path.join(toolsPath, 'splaytree.js'),
+    path.join(toolsPath, 'codemap.js'),
+    path.join(toolsPath, 'csvparser.js'),
+    path.join(toolsPath, 'consarray.js'),
+    path.join(toolsPath, 'csvparser.js'),
+    path.join(toolsPath, 'consarray.js'),
+    path.join(toolsPath, 'profile.js'),
+    path.join(toolsPath, 'profile_view.js'),
+    path.join(toolsPath, 'logreader.js'),
+    path.join(toolsPath, 'tickprocessor.js'),
+    path.join(toolsPath, 'SourceMap.js'),
+    path.join(toolsPath, 'tickprocessor-driver.js')];
+
+var tempScript = path.join(__dirname, 'tick-processor-tmp-' + process.pid);
+var cat = cp.spawn('cat', scriptFiles);
+cat.stdout.pipe(fs.createWriteStream(tempScript));
+cat.stdout.on('end', function() {
```

Comment:
@thefourtheye My understanding is that if `cat` throws an error it will manifest as a JS error in node. Regardless, this code has since changed and is no longer part of the commit in this form. I have modified the call to `spawn` to pipe all output to the parent stdio channels.